### PR TITLE
Ramatae Rework (Another Word Starting With R) [NEEDS CONFIG UPDATE]

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request
Reworks Slugcats to their new proper lore form, the Ramatae.
image
It's peam.

They now have boneless limbs, unique food prefs, are properly mute (only have sign language,) and are better at fishing/animal taming. Also lore. So much lore. Billions of lore, folks.

I also added gear harnesses because I need it for my outfit. You must understand. JUNCO megacargo pants and translation gloves are in there now too. 

This will fuck your character a bit btw.

## Why It's Good For The Game
We can't have straight up Rain World in there, can we?

## Changelog
:cl:
add: Ramatae, in their proper form!
/:cl:

### CODE REVIEW

**Comment by SylvetteSylph** [comment]
does this move ramataen ears out of the horns tab or ramataen tails out of the lizard tail tab

**Comment by honkpocket** [comment]
> does this move ramataen ears out of the horns tab or ramataen tails out of the lizard tail tab

No, `/datum/sprite_accessory/tails/lizard/ramatan` is a lizard subtype, meaning they will stay in the lizard tab.

File: code/datums/emotes.dm [hunk 1]
```diff
@@ -371,7 +371,7 @@
  */
 /datum/emote/proc/should_play_sound(mob/user, intentional = FALSE)
 	if(emote_type & EMOTE_AUDIBLE && !hands_use_check)
-		if(HAS_TRAIT(user, TRAIT_MUTE))
+		if(HAS_TRAIT(user, TRAIT_MUTE) && !isramatan(user))
```
Comment (CliffracerX) [inline code/datums/emotes.dm:374]: Doppler edit marker needed

**Review by CliffracerX — APPROVED** [review]
Singular tiny nitpick, but once it's cleared you're good for merge!

File: code/datums/emotes.dm (hunk 1 above)
Comment (Nerev4r) [reply code/datums/emotes.dm:374]: oh mb

**Pull request merged**